src/app/components/Slideout/Settings/SettingsEnableDisable.vue

```diff
@@ -13,20 +13,20 @@
 
   @Component
   export default class SettingsEnableDisable extends Vue {
-    @Prop()
+    @Prop({ required: true })
     value !: boolean | null;
```

Comment:
I think that an enum of `ALL_ON` / `SOME_ON` / `ALL_OFF` would be better than `true/false/null`.